components/board/BestArticle.tsx

```diff
@@ -49,27 +51,46 @@ function BestArticle({ article }: BestArticleProps) {
               <p className={styles["like-count"]}>{article.likeCount}</p>
             </div>
           </div>
-          <p className={styles.date}>
-            {moment(article.createdAt).format("YYYY.MM.DD")}
-          </p>
+          <p className={styles.date}>{getDate(article.createdAt)}</p>
         </div>
       </article>
     </Link>
   );
 }
 
 // NOTE - 베스트 게시글 리스트
-interface BestArticleListProps {
-  bestArticleList: Article[];
-}
 
-export default function BestArticleList({
-  bestArticleList,
-}: BestArticleListProps) {
+export default function BestArticleList() {
+  const { isDesktop, isMobile, isTablet, bestPageSizeCount } = useDeviceSize();
+  const [bestArticleList, setBestArticleList] = useState<Article[]>([]);
```

Comment:
@ssseeo0 

여기서 관리하는 데이터의 경우 누가봐도 best article이잖아요
이 경우, 그냥 도메인 이름으로만 선언해 관리하는 것도 좋습니다
```suggestion
  const [articles, setArticles] = useState<Article[]>([]);
```